azure-devops-extension/run-automator-build-task/src/automator-sdk-client.ts

```diff
@@ -0,0 +1,3 @@
+import { AutomatorSDKClient } from "@sdk/client";
```

Comment:
Do we need this file if only thing that it does is to create an instance?

Reply:
It's using the singleton module pattern. We can keep it, or we can remove it :).